connector/src/types.ts

```diff
@@ -36,12 +39,15 @@ export interface ConvictionConfigData {
 
 export interface VotingConfigData {
   id: string
-  token: TokenData
```

Comment:
Think i missed from the subgraph reviews that we remove the voting token. Where are we storing it now? Because i think it's going to be needed.